docs/cosmology/dev.rst

```diff
@@ -234,7 +234,7 @@ integrated within the code structure.
                 ...
 
 For a fully implemented example ``mypackage``, see
-https://github.com/astropy/astropy/tree/main/docs/cosmology/mypackage
+https://github.com/astropy/astropy/tree/main/astropy/cosmology/tests/mypackage
```

Comment:
@nstarman - is this correct?

Reply:
Yes, thanks! missed that in https://github.com/astropy/astropy/commit/85dc20dc62f60bd17c9b8dbc707d21b30354a207